llvm/include/llvm/ADT/DenseMap.h

```diff
@@ -496,6 +475,18 @@ class DenseMapBase : public DebugEpochBase {
   static const KeyT getTombstoneKey() { return KeyInfoT::getTombstoneKey(); }
 
 private:
+  template <typename KeyArgT, typename... Ts>
+  std::pair<iterator, bool> try_emplace_impl(KeyArgT &&Key, Ts &&...Args) {
+    BucketT *TheBucket;
```

Comment:
nit: I'd initialize this just so that there isn't any intermediate state with uninitialized values that would show up in the debugger.
```suggestion
    BucketT *TheBucket = nullptr;
```

Reply:
Updated.  Thanks!